### DISCUSSION TITLE
使用echarts 出现的问题

### DISCUSSION THREAD

**Discussion by 1169938246** [jd-opensource/micro-app#1401] [Q&A]
我项目中有使用echarts ,子仓库单独访问的时候图表可以正常显示，但是嵌入到基座里面 本地是可以的 发布到正式环境就出不来了。
![222](https://github.com/user-attachments/assets/ee169360-5841-48b9-af14-5d352624dc77)
在环境上使用基座访问
![image](https://github.com/user-attachments/assets/12acddf9-c38e-4f4e-aa2d-612f5fa14591)

**Comment by flicat** [comment]
同问，我从1.0.0-rc.8升到 1.0.0-rc.12 就不行了，后面回退回去才恢复。这应该是bug。
